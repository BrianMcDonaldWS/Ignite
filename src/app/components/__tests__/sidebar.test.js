import React from 'react';

import SideBar from '../Sidebar';
import renderToJson from './utils/renderToJson';

test('only renders content if its there', () => {
  expect(renderToJson(<SideBar />)).toMatchSnapshot();
  expect(
    renderToJson(<SideBar content={() => <h1>beans</h1>} />)
  ).toMatchSnapshot();
});
