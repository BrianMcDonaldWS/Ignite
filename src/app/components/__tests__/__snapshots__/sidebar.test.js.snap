// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`only renders content if its there 1`] = `
<div
  className="sidebar sidebar-left"
/>
`;

exports[`only renders content if its there 2`] = `
<div
  className="sidebar sidebar-left"
>
  <content
    className={undefined}
    currentPage={null}
  />
</div>
`;
